src/next-collection-view.js

```diff
@@ -558,20 +562,28 @@ const CollectionView = Backbone.View.extend({
 
   // Renders each view in children and creates a fragment buffer from them
   _getBuffer(views) {
-    const elBuffer = this.createBuffer();
+    const elBuffer = this.Dom.createBuffer();
 
     _.each(views, view => {
       renderView(view);
-      this.appendChildren(elBuffer, view.el);
+      this.Dom.appendContents({
+        el: elBuffer,
+        contents: view.el,
+        $contents: view.$el
+      });
     });
 
     return elBuffer;
   },
 
   // Override this method to do something other than `.append`.
   // You can attach any HTML at this point including the els.
-  attachHtml(collectionView, els) {
-    this.appendChildren(collectionView.el, els);
+  attachHtml(els) {
```

Comment:
Also worth mentioning.. small breaking change to `NextCollectionView`.  I really can't find a use case for passing the collectionview like this.  we're not doing it in `detachHtml` or anywhere else..  seems like now is a good time to make this very small break while it is experimental.